space-cli/versioning.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+)
+
+type scVersionResponse struct {
+	Docs []*scVersionDoc `mapstructure:"result"`
+}
+
+type scVersionDoc struct {
+	VersionNo   string `mapstructure:"version_no" json:"versionNo"`
+	VersionCode int32  `mapstructure:"version_code" json:"versionCode"`
+	//Id          TypeID     `mapstructure:"id" json:"id"`
+}
+
+//const TypeID string ="ID"
+
+func getmodule() (*cobra.Command, error) {
+
+	_ = createDirIfNotExist(getSpaceCloudDirectory())
+	_ = createDirIfNotExist(getSpaceCliDirectory())
+	_ = createFileIfNotExist(getSpaceCliConfigPath(), "{}")
+
+	//currentVersionCode := int32(0)
+	file, _ := ioutil.ReadFile("config.json")
+
+	var data scVersionResponse
+
+	_ = json.Unmarshal([]byte(file), &data)
```

Comment:
Put this inside a `readVersionConfig()` file. Don't ignore the errors. It will return a single `cliVersionDoc` object along with an error